arcgis-ios-sdk-samples/Maps/Show location history/README.metadata.json

```
@@ -0,0 +1,18 @@
+{
+    "category": "Maps",
+    "description": "This sample shows you how to display your location history on the map.",
+    "ignore": false,
+    "images": [
+        "image1.png"
+    ],
+    "keywords": [
+    ],
+    "redirect_from": "/ios/latest/swift/sample-code/show-location-history.htm",
```

FYI: There is no path from which to redirect since this is a brand new sample. This can either be the empty string or an empty array. I've been using an empty array:

```suggestion
    "redirect_from": [
    ],
```

---

I have tidied this up via [`e8d8235`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/e8d82357ea8675704803bc542acafc6859f8d3e9).